sdk/dotnet/Pulumi/Core/Options.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright 2016-2018, Pulumi Corporation
+
+#nullable enable
+
+namespace Pulumi
+{
+    public class Options

```

Comment:
```suggestion
    public class DeploymentOptions
```

???

Reply:
Have made this internal.  It may go away entirely.